lib/config.js

```diff
@@ -24,16 +24,19 @@ var fs = require("fs"),
     isAbsolutePath = require("path-is-absolute"),
     isResolvable = require("is-resolvable"),
     validator = require("./config-validator"),
-    pathIsInside = require("path-is-inside");
+    pathIsInside = require("path-is-inside"),
+    find = require("lodash.find");
```

Comment:
I'd rather use the native `Array.prototype.find` but it is not available on all Node versions we support. If we don't use this, we'd have to implement the same logic ourselves, write tests for it and maintain it. I thought, "why bother when there's already something that did all that" :)